browserstack-all.json

```diff
@@ -0,0 +1,50 @@
+{
+	"username": "BROWSERSTACK_USERNAME",
+	"key": "BROWSERSTACK_KEY",
+	"test_framework": "qunit",
+	"test_path": [
+		"test/index.html",
+		"test/startError.html",
+		"test/logs.html",
+		"test/setTimeout.html",
+		"test/amd.html",
+		"test/autostart.html",
+		"test/reporter-html/index.html"
+	],
+	"browsers": [
+		"chrome_previous",
+		"chrome_current",
+		"firefox_previous",
+		"firefox_current",
+		"opera_previous",
+		"opera_current",
+		"safari_5_1",
+		"safari_6",
+		"safari_previous",
+		"safari_current",
+		"ie_6",
+		"ie_7",
+		"ie_8",
+		"ie_9",
+		"ie_10",
+		"ie_11",
+		{
+			"os": "iOS",
+			"os_version": "6.0"
+		},
+		{
+			"os": "iOS",
+			"os_version": "7.0"
+		},
+		{
+			"os": "iOS",
+			"os_version": "8.3"
+		},
+		{
+			"os": "Windows",
+			"os_version": "10",
+			"browser": "edge",
+			"browser_version": "0.11"
```

Comment:
I wouldn't specify this if not needed; Edge build numbers may change quite frequently.